### DISCUSSION TITLE
How to import sass variables in js

### DISCUSSION THREAD

**Discussion by wanglei1020** [web-infra-dev/rsbuild#3670] [Q&A]
I am about to migrate from `ruoyi` project to `rsbuild`. The original project is based on `vue-cli@4`, which has a usage: import sass variables in js, and I'm not sure how to use that in rsbuild.

original project code : [ruoyi-vue](https://github.com/yangzongzhuan/RuoYi-Vue/blob/78bb30bb5fdd54eb34d88e3fd24098aeed29a7cc/ruoyi-ui/src/layout/components/Sidebar/index.vue#L30)

**Comment by 9aoy** [comment]
try rename `variables.scss` to `variables.module.css`. see https://rsbuild.dev/guide/basic/css-modules
